subjects/devops/flexible_function/README.md

```diff
@@ -0,0 +1,49 @@
+## create_person
+
+### Instructions
+
+Create a file `flex_function.py` that contains a function `create_person` which takes in first name and last name as required positional parameters, age and gender as positional or keyword parameters, size and job as keyword only parameters, with size and job default values: size=1.83, job="taxidermist", and returns a dictionary object like this:
+
+```python
+{
+    'first_name': 'Kevin',
+    'last_name': 'Boulin',
+    'age': 34,
+    'gender': 'male',
+    'size': 1.83,
+    'job': 'taxidermist',
+}
+```
+
+### Usage
+
```

Comment:
To make this paragraph clearer, you can add that the python lines are part of a script and, when reporting the console block you could write the following:

```console
$ python3 <name of the script>.py
<output>
$
```